atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java

```diff
@@ -99,12 +101,18 @@ public CassandraClient create() {
         try {
             return instrumentClient(getRawClientWithKeyspaceSet());
         } catch (Exception e) {
-            String message = String.format(
-                    "Failed to construct client for %s/%s", cassandraServer.proxy(), clientConfig.keyspace());
             if (clientConfig.usingSsl()) {
-                message += " over SSL";
+                throw new ClientCreationFailedException(
+                        "Failed to construct client for {}/{} over SSL",
+                        e,
+                        UnsafeArg.of("cassandraServer.proxy", cassandraServer.proxy()),
```

Comment:
cassandraServerProxy or proxy is sufficient! We don't need to name the arg the same as the variable name.

Also, what led you to it being unsafe? I haven't checked yet, just want to check your reasoning.

Reply:
It threw a compiletime error saying it was unsafe